parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -75,7 +75,7 @@ installNvidiaFabricManager() {
 
 installNvidiaContainerToolkit() {
     MARINER_NVIDIA_CONTAINER_RUNTIME_VERSION="3.13.0"
```

Comment:
Let's also remove this line since container runtime is no longer used

Reply:
Removed it.